tests/build_/test_build_request.py

```diff
@@ -1261,6 +1262,8 @@ def test_render_orchestrate_build(self, tmpdir, platforms, secret, disabled,
             assert kwargs.get('source_registry_uri') == worker_config.get_source_registry_uri()
             assert kwargs.get('sources_command') == worker_config.get_sources_command()
             assert kwargs.get('vendor') == worker_config.get_vendor()
+            assert (kwargs.get('equal_labels') or [] ==
```

Comment:
I don't see the benefit of splitting it up like that. Keep in mind that additional_kwargs doesn't necessarily have to set `equal_labels` in this test. Coupling the two would not be correct IMO.